lib/zendrive/driver.rb

```diff
@@ -29,9 +35,5 @@ def self.find(id)
     def trips(params = {})
       Trip.all({driver_id: id}, params)
     end
-
-    def score(params = {})
```

Comment:
Did this get moved or is it just no longer relevant?

Reply:
@jorgevaldivia I didn't notice this being used anywhere internally, it also overrides the `score` attr_reader.